common/src/main/scala/com/gu/media/lambda/LambdaBase.scala

```diff
@@ -10,32 +10,40 @@ import com.gu.media.aws.AwsAccess
 import com.typesafe.config.{Config, ConfigFactory}
 
 trait LambdaBase extends AwsAccess {
-  private val regionName = envSetting("REGION")
```

Comment:
These changes allow us to run the lambda locally without downloading the config from S3